assets/js/modules/analytics/components/dashboard/DashboardPopularPagesWidget.js

```diff
@@ -78,6 +83,34 @@ function DashboardPopularPagesWidget( {
 				limit: 10,
 			};
 
+			const report = store.getReport( args );
+			let pageTitles;
+
+			let hasLoadedPageTitles = true;
```

Comment:
And here, `false` by default?

Reply:
https://github.com/google/site-kit-wp/pull/3958/commits/3cddcca994d8eb6a2e939ac3aca4998081ba16f2